server/src/train/train.cpp

```diff
@@ -186,6 +188,14 @@ Train::Train(World& world, std::string_view _id) :
   Attributes::addObjectEditor(throttleSpeed, false);
   m_interfaceItems.add(throttleSpeed);
 
+  Attributes::addMinMax(accelerationRate, 0.01, 10.0);
+  Attributes::addUnit(accelerationRate, "m/s^2");
```

Comment:
Can we use a superscript two here using UTF8? see: https://github.com/traintastic/traintastic/blob/master/server/src/utils/utf8.hpp